docs/api-docs/webhooks/about-webhooks.mdx

```diff
@@ -79,6 +79,18 @@ To acknowledge a callback has been received without issue, the destination serve
 
 Need to set up a quick webhook destination URL for testing? See [Tools for Debugging and Testing Webhooks](#tools).
 
+### Response requirements
+
+BigCommerce doesn't recommend sending bodies or headers in response to webhooks. 
```

Comment:
```suggestion
BigCommerce doesn't recommend sending bodies or headers in response to webhooks. It may interfere with the successful webhook delivery acknowledgment. 
```